doc/source/whatsnew/v1.1.0.rst

```diff
@@ -407,6 +407,7 @@ Other
 - Fixed bug in :func:`pandas.testing.assert_series_equal` where dtypes were checked for ``Interval`` and ``ExtensionArray`` operands when ``check_dtype`` was ``False`` (:issue:`32747`)
 - Bug in :meth:`Series.map` not raising on invalid ``na_action`` (:issue:`32815`)
 - Bug in :meth:`DataFrame.__dir__` caused a segfault when using unicode surrogates in a column name (:issue:`25509`)
+- Timedeltas now understand µs as identifier for microsecond (:issue:`32899`)
```

Comment:
move tot he timedelta section in bug fixes, put ``us`` in double-backticks

Reply:
@jreback Done and all checks are green again